Database/SQLite/Simple.hs

```diff
@@ -307,6 +308,17 @@ execute conn template qs =
   withStatementParams conn template qs $ \(Statement stmt) ->
     void . Base.step $ stmt
 
+-- | Execute a multi-row @INSERT@, @UPDATE@, or other SQL query that is not
+-- expected to return results.
+--
+-- Throws 'FormatError' if the query could not be formatted correctly.
+executeMany :: ToRow q => Connection -> Query -> [q] -> IO ()
+executeMany conn template rows = withStatement conn template $ \stmt -> do
+    let Statement stmt' = stmt
+    forM_ rows $ \row ->
```

Comment:
Mind staying with 2 space indent instead of 4 spaces?  That's what I have tried to use throughout in these modules.